src/pages/faq.astro

```diff
@@ -42,25 +45,33 @@ const faq = await getSortedFaq();
 
     /* This is mostly copy pasted from https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#try_it */
     details {
-        border: 1px solid var(--color-semi-trans);
-        padding: 0.5em 1em 0;
-        border-radius: 6px;
+        background-color: var(--bg3);
+        padding: 0.5em 0.75em;
+        border-radius: 8px;
     }
 
     summary {
         font-weight: bold;
-        margin: -0.5em -1em 0;
-        padding: 0.5em 1em;
         cursor: pointer;
     }
 
-    details[open] {
-        padding: 0.5em 1em;
+    details[open] summary {
+        margin-bottom: 8px;
     }
 
-    details[open] summary {
-        border-bottom: 1px solid var(--color-semi-trans);
-        margin-bottom: 0.5em;
+    .faq-content {
+        background-color: var(--bgCurrentWord);
+        color: var(--grey2);
+        border-radius: 4px;
+        padding: 0.75em;
+    }
+
+    .faq-content :global(p) {
+        margin-top: 0;
+    }
+
+    .faq-content :global(*:last-child) {
```

Comment:
what is this for? From my experience selectors like this have bad performance

Reply:
removing the ugly margins on the last element of the faq content, otherwise it leaves a massive gap for no reason